extensions.json

```
@@ -105,6 +105,11 @@
       "version": "0.3.2",
       "checkout": "0.3.2"
     },
+    {
+      "id": "ccls-project.ccls",
+      "repository": "https://github.com/MaskRay/vscode-ccls",
+      "version": "0.1.10"
```

It seems fine to always publish from the default branch, so no need to pin a specific version (also, the current version seems to be `0.1.28`).

```suggestion
      "repository": "https://github.com/MaskRay/vscode-ccls"
```